baselines/exo.md

```diff
@@ -61,6 +61,7 @@ forwarding rules to exfiltrate data to external recipients.
 #### MS.EXO.1.1v1
 Automatic forwarding to external domains SHALL be disabled.
 
+<!--Policy: MS.EXO.1.1v1; Criticality: SHALL -->
```

Comment:
Not super important but we are not using all uppercase for criticallity.  I kind of like the all caps.

Reply:
I would keep criticality as is to match the other headers in the report are structured.